hw03_frequency_analysis/top.go

```diff
@@ -1,6 +1,67 @@
 package hw03frequencyanalysis
 
-func Top10(_ string) []string {
-	// Place your code here.
-	return nil
+import (
+	"sort"
+	"strings"
+	"unicode"
+)
+
+type wordstr struct {
+	Word string
+	freq int
+}
+
+func onlstr(str string) string {
+	var result []rune
+	for _, r := range str {
+		if unicode.IsLetter(r) || unicode.IsSpace(r) {
+			result = append(result, r)
+		}
+	}
+	return string(result)
+}
+
+func Top10(a string) []string {
+	a = strings.ToLower(a)
+	a = onlstr(a)
+
+	words := strings.Fields(a)
+	wordCount := make(map[string]int)
+
+	for i := 0; i < len(words); i++ {
+		if _, ok := wordCount[words[i]]; !ok {
+			wordCount[words[i]] = 1
+		} else {
+			wordCount[words[i]]++
```

Comment:
wordCount[words[i]]++ 
```suggestion
			wordCount[words[i]]++
```

Reply:
И вот этот комментарий пропустили